docs/_sidebar.md

```diff
@@ -0,0 +1,19 @@
+- Getting started
+    - [Quick start](getting-started/quickstart.md)
+- Usage
+    - Application
+        - [`.applyStateTransition()`](usage/application/applyStateTransition.md)
+        - [`.getDataContract()`](usage/application/getDataContract.md)
+        - [`.getDocuments()`](usage/application/getDocuments.md)
+        - [`.getIdentity()`](usage/application/getIdentity.md)
+    - Payment
+        - [`.getBestBlockHash()`](usage/payment/getBestBlockHash.md)
+        - [`.getBlockHash()`](usage/payment/getBlockHash.md)
+        - [`.getUTXO()`](usage/payment/getUTXO.md)
+   - Utilitaries
+        - [`.getGrpcUrl()`](usage/utils/getGrpcUrl.md)
+        - [`.getMnListDiff()`](usage/utils/getMnListDiff.md)
+        - [`.subscribeToTransactionsWithProofs()`](usage/utils/subscribeToTransactionsWithProofs.md)
+- Examples
+    - [RegisterUser](https://github.com/dashevo/dapi-client/blob/master/examples/registerUser.js)
```

Comment:
This example is terribly out of date. Remove from this until it is updated?

Reply:
Agreed, it has been removed for now :) Will wait a little bit for dapi-client to return stability and I will update this one.